set.mm

```diff
@@ -36182,14 +36182,6 @@ practical reasons (to avoid having to prove sethood of ` A ` in every use
        ( [. A / x ]. ph <-> [. A / x ]. ps ) ) $=
     ( wb wal nfa1 sp sbcbid ) ABEZCFABCDJCGJCHI $.
 
-  $( Obsolete proof of ~ sbcbi2 as of 4-May-2023.  (Contributed by Giovanni
-     Mascellani, 9-Apr-2018.)  (Proof modification is discouraged.)
-     (New usage is discouraged.) $)
-  sbcbi2OLD $p |- ( A. x ( ph <-> ps ) ->
-       ( [. A / x ]. ph <-> [. A / x ]. ps ) ) $=
-    ( wb wal cab wcel wsbc wceq abbi eleq2 sylbi df-sbc 3bitr4g ) ABECFZDACGZHZ
-    DBCGZHZACDIBCDIPQSJRTEABCKQSDLMACDNBCDNO $.
```

Comment:
changing this proof to use abbi1 saves ax-10 and ax-12 on sbcbi2
(optional, I can do it in my pr)

Reply:
good catch